package.json

```diff
@@ -26,23 +23,20 @@
     "@ctrl/ts-base32": "^1.2.4",
     "@types/app-root-path": "^1.2.4",
     "@types/cors": "^2.8.8",
-    "@types/sodium-native": "^2.3.5",
     "@types/sqlite3": "^3.1.6",
-    "ajv": "^6.12.6",
     "argparse": "^2.0.1",
     "assert": "^2.0.0",
     "axios": "^0.19.2",
+    "boa-sdk-ts": "https://github.com/MichaelKim20/boa-sdk-ts.git",
```

Comment:
If https://github.com/bpfkorea/boa-sdk-ts/pull/54 is merged first, I will modify this.

Reply:
It will need to

Reply:
I changed to "boa-sdk-ts": "git+https://github.com/bpfkorea/boa-sdk-ts.git"